### PR TITLE
ldid: linux compatibility

### PR DESCRIPTION
**Pull request by 0x2b3bfa0**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/ldid.rb [hunk 1]
```diff
@@ -16,30 +16,35 @@ class Ldid < Formula
     sha256 cellar: :any, mojave:         "986774410fa97d8f9afc40d378ee97c9ea6e3b18d8055f3881c12a68466e38b6"
   end
 
+  depends_on "go" => :test
   depends_on "libplist"
-  depends_on :macos
+  depends_on "libxml2"
   depends_on "openssl@1.1"
```
Comment (carlocab) [inline Formula/ldid.rb:22]:
```suggestion
  depends_on "openssl@1.1"
  uses_from_macos "libxml2"
```

**Comment by carlocab** [comment]
We don't need `go` here if all we're looking for is a MachO binary. Fixed that up for you.

**Comment by 0x2b3bfa0** [comment]
Thanks! I skimmed through the [“add a test to the formula”](https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula) section and didn't notice the [standard test fixtures](https://github.com/Homebrew/brew/tree/master/Library/Homebrew/test/support/fixtures) bullet point. 🤦🏼‍♂️

**Review by carlocab — APPROVED** [review]
Thanks! A Linux build that you can `brew install` will be uploaded shortly. (You may need to `brew update` first.)

**Comment by carlocab** [comment]
Available starting d275a915e1771146a00dcb9df4c31078e46b0f6c. Thanks for your contribution!

**Comment by 0x2b3bfa0** [comment]
Thank you very much for all your help! 🙏🏼

**Pull request merged**